ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/remote/HttpJsonConnectorExecutorTest.java

```diff
@@ -59,6 +63,86 @@ public void setUp() {
         MockitoAnnotations.openMocks(this);
     }
 
+    @Test
+    public void invokeRemoteModel_GetMethodSuccessPath() throws ClientProtocolException, IOException {
+        try {
+            ConnectorAction predictAction = ConnectorAction.builder()
+                .actionType(ConnectorAction.ActionType.PREDICT)
+                .method("get")
```

Comment:
Looks like difference between `invokeRemoteModel_GetMethodSuccessPath` & `invokeRemoteModel_POSTMethodSuccessPath` is only here. May be we can use a function which could be reused to save lines?